config/v1/types_feature.go

```diff
@@ -114,6 +114,7 @@ var FeatureSets = map[FeatureSet]*FeatureGateEnabledDisabled{
 		with("CSIMigrationvSphere").         // sig-storage, fbertina, Kubernetes feature gate
 		with("ExternalCloudProvider").       // sig-cloud-provider, jspeed, OCP specific
 		with("CSIDriverSharedResource").     // sig-build, adkaplan, OCP specific
+		with("CSIDriverGCPFilestore").       // sig-storage, fbertina, OCP specific
 		with("BuildCSIVolumes").             // sig-build, adkaplan, OCP specific
```

Comment:
Why are we specifying this here? This will be an optional operator installable via OLM and hence TP process for those operators are kinda different. I don't think we need to carry this here.

Reply:
Good catch, removed.